examples/resources/katapult_virtual_machine_group/resource.tf

```diff
@@ -0,0 +1,10 @@
+# Create a virtual machine group
+resource "katapult_virtual_machine_group" "web-1" {
+  name = "vm group"
+}
+
+# Create a segregated virtual machine group
```

Comment:
These examples need updating, as `segregate` now defaults to `true` rather than `false`.